quickwit/quickwit-metastore/src/metastore/file_backed/mod.rs

```diff
@@ -379,22 +379,54 @@ impl FileBackedMetastore {
     /// Returns the list of splits for the given request.
     /// No error is returned if any of the requested `index_uid` does not exist.
```

Comment:
there is no mention about the expected order, yet it seems like the code takes great care to respect some very specific order.